l10n_es_facturae_service/models/sale_order.py

```diff
@@ -16,5 +16,7 @@ def _prepare_invoice_line(self, **optional_values):
             facturae_file_reference=order.insured_ident_cardnum,
             facturae_receiver_transaction_reference=order.policy_number,
             facturae_receiver_contract_reference=order.auth_number,
-            facturae_receiver_transaction_date=order.service_date,
+            facturae_receiver_transaction_date=fields.Date.context_today(
+                order, order.service_date
+            ),
```

Comment:
Is this getting the TZ from the user who created the order? It seems it's the current user. Please check it out